services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/NeutronRouterServiceImpl.java

```diff
@@ -280,13 +294,14 @@ public RouterInterfaceResponse removeAnInterfaceToNeutronRouter(String projectid
             }
 
             List<RouteEntry> routeEntities = routeTable.getRouteEntities();
-            for (RouteEntry routeEntry : routeEntities) {
-                String nextHop = routeEntry.getNexthop();
-                if (gatewayIp.equals(nextHop)) {
-                    throw new RouterInterfaceAreUsedByRoutes();
+            if (routeEntities != null) {
```

Comment:
Let us print one extra line of comment when routeEntities == null.

Reply:
@xieus comments added.